modules/actors/actor-sheet-ffg.js

```diff
@@ -1113,20 +1113,21 @@ export class ActorSheetFFG extends ActorSheet {
    * Update specialization talents
    * @param  {Object} data
    */
-  async _updateSpecialization(data) {
+  async _updateSpecialization(data, new_talent_list) {
```

Comment:
The `new_talent_list` object doesn't seem to get used in the method... do we need it?

Reply:
```suggestion
  async _updateSpecialization(data) {
```
If not...